10up/action-pr-helper
# Added support for passing pull request validation regex from workflow file.

### Description of the Change
PR adds support for passing PR description, changelog, and credits validation regex and the possibility of disabling the validation of it.


Closes #76 

### How to test the Change
1. Add workflow to the repository using this action @ PR branch. (you can use any repo for the test)
1. Verify that the `description`, `changelog`, and `credits` validation working fine. (Without adding any input in the workflow file)
1. Pass any custom regex to `description-validation`, `changelog-validation`, and `credit-validation` inputs in the workflow file and make sure the action validates PR accordingly.
1. Pass `false` to all above validation inputs and make sure PR validation get disabled.


### Changelog Entry
> Added - Support for passing pull request validation regex from workflow file.


### Credits
Props @senhordaluz @iamdharmesh 


### Checklist:
